src/plugins/intel_gna/src/gna_graph_tools.hpp

```diff
@@ -21,6 +21,10 @@ namespace InferenceEngine {
 
 static constexpr size_t invalid_data_idx = std::numeric_limits<size_t>::max();
 
+static bool DoNotSkip(CNNLayerPtr layer) {
```

Comment:
why function as static in header? This will lead to copy of the function in each place where header will be included. This lead to increase of executable size AFAIK

Reply:
By mistake.